package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "nodejs-mentoring",
+  "version": "1.0.0",
+  "description": "EPAM node.js mentoring Excercises and tasks ",
+  "type": "module",
+  "scripts": {
+    "start": "node -v && node main.js",
+    "dev": " node main.js",
```

Comment:
Use either `nodemon` or `node --watch main.js` for `dev` mode. These options will track your changes on the files while the server is still running, and refresh on any change without stopping/starting the server to save time.
Note: use only for dev/test, not for prod.
Suggestion: read about this mechanism

Reply:
As nodemon is the task runner I didn't feel comfortable adding it to the scripts.
The only reason I have both because of the following line: "index.js file is started via npm script using nodemon". My thinking was that based on the quoted line the following order should be established index.js -> npm script -> nodemon. The script name `start` will be run by a `npx nodemon` command by default that is why its not included in the scripts.

My understanding was that noodemon is usually a global dependency so one can execute multiple scripts with the same version of the task runner.

For `--watch` I dont usually add it to my dev scrips because I am using a high rate of autosave in my IDEs so the constant reruns would be just distracting.

Reply:
BTW your method also works so no issue here, just wanted to clarify the task was to add to scripts.